src/com/company/BiggestNumber.java

```diff
@@ -0,0 +1,91 @@
+package com.company;
+
+import java.util.Scanner;
+
+/**
+ * from the 3 numbers provided by user returns the biggest number of the 3
+ * @author Marius Cristea
+ * @version 1
+ */
+public class BiggestNumber {
+    String firstNumber;
+    String secondNumber;
+    String thirdNumber;
+
+    /**
+     * contructor of the class
+     */
+
+    public BiggestNumber(){
+        System.out.println("Insert first number");
+        firstNumber = createString();
+        System.out.println("Insert second number");
+        secondNumber = createString();
+        System.out.println("Insert third number");
+        thirdNumber = createString();
+        returnBiggestNumber(firstNumber,secondNumber,thirdNumber);
+    }
+
+    public static void main (String[] args){
```

Comment:
try to avoid unused code. When you created the menu you should have completely removed the main methods from the other classes.